tensorflow/core/util/tensor_bundle/byte_swap.h

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef BYTE_SWAP_H
```

Comment:
#ifndef header guard has wrong style, please use: TENSORFLOW_CORE_UTIL_TENSOR_BUNDLE_BYTE_SWAP_H_

Reply:
Switched to `TENSORFLOW_CORE_UTIL_TENSOR_BUNDLE_BYTE_SWAP_H_`